config/connectors/definitions/mongodb-forgerock-data.json

```diff
@@ -0,0 +1,253 @@
+{
+	"_id": "provisioner.openicf/ForgerockData",
+	"connectorRef": {
+		"connectorHostRef": "chs-group",
```

Comment:
Can you change this to "forgerock-export"? That's the new remote server that will be configured to use the new RCS instance

Reply:
done